README.md

```diff
@@ -79,6 +79,39 @@ governance](https://pyodide.org/en/stable/project/governance.html).
 - Stack Overflow: [stackoverflow.com/questions/tagged/pyodide](https://stackoverflow.com/questions/tagged/pyodide)
 - Discord: [Pyodide Discord](https://dsc.gg/pyodide)
 
+## Sponsors
+
+### Institutional Support
+
+Pyodide has received support from various institutions, grants, and employers over the years.
+
+For a full list of current and historical sponsors, please see the [Funding](https://pyodide.org/en/stable/project/about.html#funding) section of our About page.
+
+If you’re interested in supporting Pyodide, check out our [OpenCollective](https://opencollective.com/pyodide) and [GitHub Sponsors](https://github.com/sponsors/pyodide) pages.
+
+### Sponsors
```

Comment:
I think if we're going to put the big institutional supporters who each provided or are providing upwards of $50,000 worth of support in a separate funding page, we shouldn't include these much smaller sponsors directly in the readme. So I'd say move this section into about.html#funding too?